### PR TITLE
chore(sync-service): Label processes that handle requests

### PR DESCRIPTION
**Pull request by robacourt**

### CODE REVIEW

File: packages/sync-service/lib/electric/plug/label_process_plug.ex [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    plug :match
+    plug Electric.Plug.LabelProcessPlug
+  """
+
+  def init(opts), do: opts
+
+  def call(conn, _opts) do
+    conn
+    |> process_label()
```
Comment (icehaunter) [inline packages/sync-service/lib/electric/plug/label_process_plug.ex:16]: Maybe pull request ID from the conn too and use that in the label as well? Generally all logs ought to be correlatable by the request ID, not by full path, right?

**Review by icehaunter — APPROVED** [review]
GJ, just one question

File: packages/sync-service/lib/electric/plug/label_process_plug.ex (hunk 1 above)
Comment (robacourt) [reply packages/sync-service/lib/electric/plug/label_process_plug.ex:16]: Ooh, good idea. I'll add it.

Comment (robacourt) [reply packages/sync-service/lib/electric/plug/label_process_plug.ex:16]: Done

**Pull request merged**